README.md

```diff
@@ -159,43 +164,44 @@ DISCLAIMER: All below async references were added taking into consideration only
 	2. [Pandas](https://pandas.pydata.org/docs/user_guide/10min.html) - fast, powerful, flexible and simple data analysis & manipulation library
 	3. [Scipy](https://scipy.org/) - a scientific computation library that uses NumPy underneath
-12. <u>App Settings / Configuration</u> - representation and access to application settings & configurations, e.g. connection strings, services URLs, anything that shouldn't be hard coded and should be accessible to system code in a configurable manner.
+13. <u>App Settings / Configuration</u> - representation and access to application settings & configurations, e.g. connection strings, services URLs, anything that shouldn't be hard coded and should be accessible to system code in a configurable manner.
 	1. Extensive support using [_dynaconf_](https://www.dynaconf.com/) supporting multi-environment, many formats, external config stores (e.g. Redis), unit tests and more
 	2. Basic support from [_pydantic_](https://pydantic-docs.helpmanual.io/usage/settings/) library above, can be extended to support [multi-environment](https://rednafi.github.io/digressions/python/2020/06/03/python-configs.html) and custom loaders can make it leverage _dynaconf_
-13. <u>File System</u>
+14. <u>File System</u>
 	1. file system access using async IO - [_aiofiles_](https://pythonrepo.com/repo/Tinche-aiofiles-python-files)
 	2. Path object: [Real Python reference](https://realpython.com/python-pathlib/)
-14. <u>Http Client</u> - sending HTTP requests using async IO, popular alternatives:
+15. <u>Http Client</u> - sending HTTP requests using async IO, popular alternatives:
 	1. [_httpx_](https://www.python-httpx.org/)
 		1. Supports: HTTP/2, client certificate, full request & response hooks, env variables config, [OAuth2 extension](https://colin-b.github.io/httpx_auth/)
 		2. Doesn't support: websockets
 	2. [_aiohttp_](https://docs.aiohttp.org/en/stable/)
 		1. Supports: web sockets, client certificate, partial request & response hooks (allowed modifications: requests headers, enables Authentication flows), OAuth2 ([via another package](https://docs.authlib.org/en/v0.12.1/client/aiohttp.html))
 		2. Doesn't support: HTTP/2, env variables config
-	* Recommended _httpx_ for richer feature set, for web sockets client use [_websockets_](https://websockets.readthedocs.io/en/stable/) 
-15. <u>SQL ORM</u> - object relational mapper for working with SQL databases - [_SQLAlchemy_](https://www.sqlalchemy.org/)
+	* Recommended _httpx_ for richer feature set, for web sockets client use [_websockets_](https://websockets.readthedocs.io/en/stable/)
+	3. ([Contributed by HomiGrotas](https://github.com/HomiGrotas)) Requests -  The requests module allows you to send HTTP requests using Python [_Official docs reference_](https://requests.readthedocs.io/en/latest/user/quickstart/)<br>***Note: The requests module is synchronous module and therefore it's not recommended for production code expected to be scalable, but can be useful for other scenarios.** 
```

Comment:
Can you please move the Requests module above the recommendation to use httpx?

Also, isn't is usually called "requests" instead of "Requests"? That's how I always see it

Reply:
The formal name is Requests, as in the project documentation.